src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Color {
+    Red,
+    Green,
+    Blue,
+    Yellow,
+    Orange
```

Comment:
Wrong case for constants
```suggestion
    Red,
    Green,
    Blue,
    Yellow,
    Orange
```